website/source/api/health.html.markdown

```diff
@@ -61,7 +61,8 @@ $ curl \
     "Notes": "",
     "Output": "",
     "ServiceID": "",
-    "ServiceName": ""
+    "ServiceName": "",
+    "ServiceTags": null
```

Comment:
That's a good question. We've been dealing with this at the agent response level with code like this -
 https://github.com/hashicorp/consul/blob/v0.8.1/command/agent/catalog_endpoint.go#L79-L82, though I think there are a couple cases that do it in the RPC response. It really does seem like a JSON/UX concern more than anything that we should have to think about inside Consul's core. A while back I did a quick look to see if we could handle this in the JSON encoder, but there didn't seem to be an easy way to hook this in there, and some folks discouraged it, iirc. That _seems_ like the right place to do it, though, since basically everything goes through one place for serialization - https://github.com/hashicorp/consul/blob/v0.8.1/command/agent/http.go#L387-L396.